tests/codegen/call_size.cpp

```diff
@@ -0,0 +1,14 @@
+#include "common.h"
+
+namespace bpftrace {
+namespace test {
+namespace codegen {
+
+TEST(codegen, call_size)
+{
+  test("BEGIN { @x[1] = 1; } kprobe:f { printf(\"%d\", size(@x)); }", NAME);
```

Comment:
```suggestion
  test("BEGIN { @x[1] = 1; } kprobe:f { @y = size(@x)); }", NAME);
```
Not a big deal, but it makes the LLVM IR easier to read if the test cases are doing the bare minimum they need to. The printf adds a bunch of extra basic blocks to handle ring buffer statistics. If there's a way to test this without even assigning size to a map then that'd be even better!

If we can keep the tests cases smaller and more focused then they'll hopefully only be affected by smaller parts of codegen, so won't get updated so often and we'll have an easier time reading through the diffs for suspicious IR changes.

Reply:
This is a good suggestion.

> If there's a way to test this without even assigning size to a map then that'd be even better!

It seems that
```suggestion
  test("BEGIN { @x[1] = 1; } kprobe:f { $s = size(@x); }", NAME);
```
is ~10 instructions shorter, so I'll use that.